tutorial/part2/chapter6.md

```diff
@@ -90,101 +92,93 @@ For this reason, we recommend importing the data before initialising the app, ri
+In this example, we have accessed the data (Excel file) outside the folder where our main app is located; therefore, we specified an absolute filepath (also called "the full path"). The path is saved as a raw string (hence the "r" just before the string containing the path. This is done because VS Code may trigger a warning when using a normal string).
 
-If the data file is located directly inside the folder where your main app file is (or within the VS Code working directory), the full path is not required; you may only specify the filename. The code will then become: 
+If the data file is located directly inside the folder where your main app file is (or within the VS Code working directory), the full path is not required; you may only specify the filename (also called "the relative path"). The code will then become:
 
 ```
-import pandas as pd
-filepath = 'data_01.xlsx' 
+filepath = 'data_01.xlsx'
 df1 = pd.read_excel(filepath, sheet_name='Sheet1')
-print(df1)
 ```
 
+If we would peak at the data we read into the dataframe `df1`, it would look like the following:
+
+![Excel data 01](./ch6_files/data01.JPG)
+
 ```{note}
 If you experience any difficulties in finding your filepath, please check [this screenshot](https://github.com/open-resources/dash_curriculum/blob/main/tutorial/part2/ch6_files/Helper01.JPG), showing how to find the filepath directly from VS Code.
 ```
+
 ```{tip}
 In production versions, when apps get deployed, the best practice is to have a "data" folder, where all data files are stored and therefore accessed by the app code. This will be covered in [Chapter 14](https://open-resources.github.io/dash_curriculum/part4/chapter14.html).
 ```
 
-We uploaded one Excel tab (named "Sheet1") to a data frame called "df1". After the data uploads, the dataframe would look like the following:
-
-![Excel data 01](./ch6_files/data01.JPG)
-
 #### CSV files
-We will now upload the same data from above, but from a .csv file named [data_02](https://github.com/open-resources/dash_curriculum/blob/main/tutorial/part2/ch6_files/data_02.csv). Please follow the link and click the "Copy raw contents" button (next to the pencil edit button), then paste the records into a new Notepad file (Windows) or TextEdit file (MacOS), and save it as "data_02.csv". Find the path of your file to update the `filename` in the code below.
+
+We will now read in the same data from above, but from a .csv file named [data_02](https://github.com/open-resources/dash_curriculum/blob/main/tutorial/part2/ch6_files/data_02.csv). Please follow the link and click the "Raw" button in the top right corner of the file, right click the new page that opened and select "Save page as"/"Save as" and save it as "data_02.csv". Find the path of your file to update the `filename` in the code below.
 
 ```
-import pandas as pd
 filepath = r'C:\Users\User1\Downloads\data_02.csv'
 col_names = ['country','continent','year','pop']
 df2 = pd.read_csv(filepath, sep='|', usecols=col_names)
```

Comment:
I am hesitant to introducing these new options `sep` and `usecols` while also introducing a new way of reading files. I think we should read the file in the simplest way and then we could mention that there are also additoinal options and briefly what they do with a link instead of showing them as the default way of reading from a CSV